modules/plymouth/theme-script.nix

```diff
@@ -0,0 +1,178 @@
+{
+  lib,
+  config,
+  ...
+}:
+let
+  cfg = config.stylix.targets.plymouth;
+
+  backgroundColor =
+    with config.lib.stylix.colors;
+    "${base00-dec-r}, ${base00-dec-g}, ${base00-dec-b}";
+
+  foregroundColor =
+    with config.lib.stylix.colors;
+    "${base05-dec-r}, ${base05-dec-g}, ${base05-dec-b}";
+in
+builtins.toFile "stylix-plymouth-theme" ''
```

Comment:
Why is this PR causing every module's CI to fail:

```
error: cannot build '/nix/store/0mqqdfdq98gjywsizcs8fabmfpnsx1hy-base16-schemes-unstable-2024-11-12.drv^out' during evaluation because the option 'allow-import-from-derivation' is disabled
```

Can you try rebasing on top of master again? Maybe that resolves it.

Reply:
> Can you try rebasing on top of master again? Maybe that resolves it.

Yes, this should work. I had this issue in one of my own PRs. The branch was created before d8289c3f0e5995863921ea207392c122f5d59f6d but pushed more recently, therefore GitHub is running the new CI.